sidebarsAstro.js

```diff
@@ -92,7 +92,16 @@ module.exports = {
         "configure-worker-queues",
         "api-keys",
         "environment-variables",
-        "secrets-backend",
+        {
+          type: "category",
+          label: "Configure a secrets backend",
```

Comment:
```suggestion
          label: "Set up a secrets backend",
```
Since we just removed a bunch of "configure"/ "manage" instances from the sidebar, thought it would be better to have a more descriptive word here. Also makes it consistent with the doc titles.